ec2/spark_ec2.py

```diff
@@ -87,7 +90,10 @@
     "1.3.1": "0.5.0",
     "1.4.0": "0.6.4",
     "1.4.1": "0.6.4",
-    "1.5.0": "0.7.1"
+    "1.5.0": "0.7.1",
+    "1.5.1": "0.7.1",
+    "1.5.2": "0.7.1",
+    "1.6.0": "0.7.1",
```

Comment:
I'm not sure about this mapping. I'm just assuming that a version of Tachyon that works with 1.5.0 will also work with later versions. @haoyuan?